### PR TITLE
refactor(properties): set properties on EquipmentModificationEntity

### PR DESCRIPTION
**Pull request by TheMaskedTurtle**

### CODE REVIEW

**Review by klesaulnier — COMMENTED** [review]
Also, change "random_uuid()" function as discussed

**Review by klesaulnier — APPROVED** [review]
Test OK
Just need to check N+1 query and to check if it's ok to make 2 different SQL file in order to pass tests

File: src/main/java/org/gridsuite/modification/server/entities/equipment/modification/EquipmentModificationEntity.java [hunk 1]
```diff
@@ -37,5 +43,23 @@ public void update(ModificationInfos modificationInfos) {
 
     private void assignAttributes(EquipmentModificationInfos equipmentModificationInfos) {
         equipmentId = equipmentModificationInfos.getEquipmentId();
+        List<FreePropertyEntity> newProperties = equipmentModificationInfos.getProperties() == null ? null :
+            equipmentModificationInfos.getProperties().stream()
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/modification/server/entities/equipment/modification/EquipmentModificationEntity.java:47]: Add a method `FreePropertyInfos::toEntity`

File: src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java [hunk 2]
```diff
@@ -803,7 +870,7 @@ public void testStatusLineModification() {
 
         SQLStatementCountValidator.reset();
         networkModificationRepository.deleteModificationGroup(TEST_GROUP_ID, true);
-        assertRequestsCount(2, 0, 0, 3);
+        assertRequestsCount(7, 0, 0, 3);
```
[the remark this answers is not in the document]
Comment (SlimaneAmar) [reply src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java:873]: Yes on select
Need to investigate

**Review by SlimaneAmar — CHANGES_REQUESTED** [review]

File: src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (SlimaneAmar) [reply src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java:873]: The JPA cascade delete does a select for each relation
 The refactor of entities deletion will correct this

File: src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java [hunk 3]
```diff
@@ -701,21 +705,84 @@ public void testSubstationCreation() {
...
-        assertRequestsCount(2, 0, 0, 3);
+        assertRequestsCount(3, 0, 0, 3);
 
         assertThrows(new NetworkModificationException(MODIFICATION_GROUP_NOT_FOUND, TEST_GROUP_ID.toString()).getMessage(),
             NetworkModificationException.class, () -> networkModificationRepository.getModifications(TEST_GROUP_ID, false, true)
         );
     }
 
+    @Test
+    public void testSubstationCreationWithProperties() {
```
Comment (SlimaneAmar) [inline src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java:724]: Move these tests into `SubstationCreationTest` and `SubstationModificationTest` classes

File: src/main/java/org/gridsuite/modification/server/entities/equipment/modification/EquipmentModificationEntity.java (hunk 1 above)
Comment (TheMaskedTurtle) [reply src/main/java/org/gridsuite/modification/server/entities/equipment/modification/EquipmentModificationEntity.java:48]: done

File: src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java (hunk 3 above)
Comment (TheMaskedTurtle) [reply src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java]: finally I removed them properties are already properly tested in `SubstationCreationTest` and `SubstationModificationTest`

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**